src/sql/azdata.d.ts

```diff
@@ -41,6 +41,8 @@ declare module 'azdata' {
 
 		export function registerCapabilitiesServiceProvider(provider: CapabilitiesProvider): vscode.Disposable;
 
+		export function registerSqlAssessmentServicesProvider(provider: SqlAssessmentServicesProvider): vscode.Disposable;
```

Comment:
Changes should go in azdata.proposed.d.ts first

(make sure they're namespaced correctly when you do so)